pkg/flags/logging.go

```diff
@@ -31,7 +31,7 @@ import (
 )
 
 type LoggingConfig struct {
-	featureGate featuregate.MutableFeatureGate
+	featureGate featuregate.MutableVersionedFeatureGate
```

Comment:
This seems unrelated to this PR

Reply:
Got this build error:
```
17.56 # github.com/NVIDIA/k8s-dra-driver-gpu/pkg/flags
17.56 pkg/flags/logging.go:45:43: cannot use l.featureGate (variable of interface type featuregate.MutableFeatureGate) as featuregate.MutableVersionedFeatureGate value in argument to logsapi.AddFeatureGates: featuregate.MutableFeatureGate does not implement featuregate.MutableVersionedFeatureGate (missing method AddVersioned)
```

I think that was because of `k8s.io/component-base` v032.0 -> v0.33.0.

`logsapi.AddFeatureGates(l.featureGate)` below is documented here: https://godocs.io/k8s.io/component-base/logs/api/v1#AddFeatureGates, in 0.33 it expects type `MutableVersionedFeatureGate`.